Adds documentation for new external login provider functionality and cleans up a bit of the existing docs (#2788)
umbraco/UmbracoDocs: Reference/Security/index-v7.md

### [BackOfficeUserManager](backoffice-user-manager-v7.3.0.md) and Events

The [`BackOfficeUserManager`](backoffice-user-manager.md) is the ASP.NET Identity [UserManager](https://docs.microsoft.com/en-us/previous-versions/aspnet/dn613290(v=vs.108)) implementation in Umbraco. It exposes APIs for working with Umbraco User's via the ASP.NET Identity including password handling.

### Custom password check

In most cases [External login providers (OAuth)](external-login-providers.md) will meet the needs of most users when needing to authenticate with external resources but in some cases you may need to only change how the username and password credentials are checked.

This is typically a legacy approach to validating credentials with external resources but it is possible.

You are able check the username and password against your own credentials store by implementing a [`IBackOfficeUserPasswordChecker`](custom-password-checker-v7.3.0.md).

#### Authenticating with Active Directory credentials

> If you are using a network based Azure Directory (not Azure Active Directory), we have setup a guide on how to [connect the backoffice to Active Directory](Authenticate-With-AD-vpre8.md). It should be pretty straight forward with the `ActiveDirectoryBackOfficeUserPasswordChecker`.

Reviewer: ```suggestion
If you are using a network-based Azure Directory (not Azure Active Directory), we have set up a guide on how to [connect the backoffice to Active Directory](Authenticate-With-AD-vpre8.md). It can be done using the  `ActiveDirectoryBackOfficeUserPasswordChecker`.
```
